src/controller/controller.cpp

```diff
@@ -28,6 +28,7 @@ long Controller::run() {
 
 bool Controller::simulate() {
     terminalview.render(world);
+    if(sdlview.render(world)) return false;
```

Comment:
also, please add a switch-case statement for what should be rendered, or give the controller only one View object anyways.

Also, I'd suggest you make terminalView and sdlView a subclass of a common abstract View class which provides the pure virtual method `virtual bool render (const World & world) const = 0;`

Reply:
doesn't it make sense to have both the terminal view and the sdl view?  

Also subclassing doesn't really have a point imo since they really don't have anything in common but the "render" name

Reply:
Use if clauses instead of switch, with constants defined in config.h, as already used in controller.cpp: ```if (RENDER_TERMINALVIEW) terminalview.render(world);```